openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx

```diff
@@ -15,19 +15,26 @@
   * limitations under the License.
 */
 
+import classNames from 'classnames';
 import React, { useEffect, useState } from 'react';
 import ReactMarkdown from 'react-markdown';
 import rehypeRaw from 'rehype-raw';
 import gfm from 'remark-gfm';
 
 /*eslint-disable  */
-const RichTextEditorPreviewer = ({ markdown }: { markdown: string }) => {
+const RichTextEditorPreviewer = ({
+  markdown,
+  className,
```

Comment:
@ShaileshParmar11 here className is optional so we should give it default value as `className='' `